dartagnan/src/test/java/com/dat3m/dartagnan/program/analysis/alias/AliasTest.java

```diff
@@ -0,0 +1,74 @@
+package com.dat3m.dartagnan.program.analysis.alias;
+
+import com.dat3m.dartagnan.configuration.Arch;
+import com.dat3m.dartagnan.parsers.cat.ParserCat;
+import com.dat3m.dartagnan.parsers.program.ProgramParser;
+import com.dat3m.dartagnan.program.Program;
+import com.dat3m.dartagnan.verification.Context;
+import com.dat3m.dartagnan.verification.VerificationTask;
+import com.dat3m.dartagnan.wmm.Relation;
+import com.dat3m.dartagnan.wmm.Wmm;
+import com.dat3m.dartagnan.wmm.analysis.RelationAnalysis;
+import org.junit.Test;
+import org.sosy_lab.common.configuration.Configuration;
+import org.sosy_lab.common.configuration.InvalidConfigurationException;
+
+import java.io.File;
+import java.util.EnumSet;
+import java.util.LinkedList;
+import java.util.List;
+
+import static com.dat3m.dartagnan.configuration.OptionNames.ENABLE_EXTENDED_RELATION_ANALYSIS;
+import static com.dat3m.dartagnan.configuration.Property.PROGRAM_SPEC;
+import static com.dat3m.dartagnan.utils.ResourceHelper.getRootPath;
+import static com.dat3m.dartagnan.utils.ResourceHelper.getTestResourcePath;
+import static com.dat3m.dartagnan.verification.solving.ModelChecker.*;
+import static org.junit.Assert.assertEquals;
+
+public class AliasTest {
+
+    private final String testPath = getTestResourcePath("libvsync/bounded_mpmc_check_empty-opt.ll");
+    private final String modelPath = "cat/c11.cat";
+    private final Arch target = Arch.C11;
+
+    @Test
+    public void test() throws Exception {
+        Program program = new ProgramParser().parse(new File(testPath));
+        Wmm wmm = new ParserCat().parse(new File(getRootPath(modelPath)));
+        Configuration baseConfig = Configuration.builder().build();
+        VerificationTask baseTask = createTask(program, wmm, baseConfig);
+        wmm.configureAll(baseTask.getConfig());
+        preprocessProgram(baseTask, baseTask.getConfig());
+        preprocessMemoryModel(baseTask, baseTask.getConfig());
+
+        Relation loc = wmm.getRelations().stream().filter(r -> r.getNameOrTerm().equals("loc")).findAny().orElseThrow();
+        Configuration config = Configuration.builder()
+                //.setOption(ALIAS_METHOD, FIELD_SENSITIVE.asStringOption())
+                //.setOption(ALIAS_METHOD, FIELD_INSENSITIVE.asStringOption())
+                .setOption(ENABLE_EXTENDED_RELATION_ANALYSIS, "false")
+                .build();
+
+        List<Integer> sizes = new LinkedList<>();
+        for (int i = 0; i < 10; i++) {
+            Context nativeContext = Context.create();
+            VerificationTask nativeTask = createTask(program, wmm, config);
+            performStaticProgramAnalyses(nativeTask, nativeContext, nativeTask.getConfig());
+            performStaticWmmAnalyses(nativeTask, nativeContext, nativeTask.getConfig());
+            RelationAnalysis nativeRa = nativeContext.get(RelationAnalysis.class);
+            System.out.println(nativeRa.getKnowledge(loc).getMaySet().size());
```

Comment:
Remove

Reply:
Actually, I'd like to remove this entire test class again, because it cannot prove determinism.  Should it fail in a future PR, it might be caused by some already merged change.  It was helpful for this PR, though.

Reply:
While it cannot prove determinism, it can find bugs when the implementation is not (and it might catch bug if the analysis is modified again in the future). Thus, I see zero advantage of getting rid of the test class.